src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
 
+    public static final DateTimeFormatter DATE_TIME_FORMATTER
```

Comment:
if we use this variable only inside this class let' make it private

Reply:
done